spec/rails_helper.rb

```diff
@@ -40,7 +41,11 @@ def create_items
   end
 
   def create_user
-    let!(:user_one) { User.create(username: 'rossedfort', password: 'password') }
+    let!(:user_one) { User.create(username: 'rossedfort', password: 'password', role: 0) }
+  end
+
+  def create_admin
+    let!(:admin_one) { User.create(username: 'admin', password: 'admin', role: 1) }
```

Comment:
Line is too long. [83/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.